### DISCUSSION TITLE
Software only available in Docker - How should we do them?

### DISCUSSION THREAD

**Discussion by quantumryuu** [community-scripts/ProxmoxVE#581] [Q&A]
Hello all!
I've seen some requests that are up at the Project Board that seem to work only with docker, for example [Komodo](https://github.com/orgs/community-scripts/projects/9/views/1?pane=issue&itemId=88860919) and [KeepassXC](https://github.com/orgs/community-scripts/projects/9/views/1?filterQuery=keepass&pane=issue&itemId=88867498).

Should they be created like the script of [Dockge](https://github.com/community-scripts/ProxmoxVE/blob/main/install/dockge-install.sh)?

**Comment by MickLesk** [comment]
Scripts that are only available for docker, need to build into bare Metal. Like adventurelog for example.

I thought, at least that's how I understood it, that Komodo is something like portainer, i.e. a management platform? That's actually the reason why I said okay, then you have the choice between docker (portainer or compose), dockge and Komodo
Correct me if I'm wrong. Then I'll have to look at it again and revoke the approval if necessary.

**Reply by quantumryuu** [reply]
From what I understand, Komodo, is a CI/CD tool mainly focused on docker(?)
I found the following in the codes:
To run Komodo, you will need Docker. See [the docker install docs](https://docs.docker.com/engine/install/).

Could not find any binary etc so we create the install script, it will have to be like dockge.
Install docker, pull docker-compose.yml etc etc

**Reply by Mellowlynx** [reply]
As I think, a home lab is about tinkering and trying stuff out on your own to learn.
We should not add those scripts. Most of the time there is an installation example (docker compose file) and it's just posting that to deploy the app.
The helper part is the Docker LXC, and from there the user should be good on its own.

**Reply by quantumryuu** [reply]
Then we should review the script requests cause I found like 3-4 until now that are available only via docker.

**Reply by MickLesk** [reply]
Which ones? The most of our Scripts are based on docker :smile:

**Reply by quantumryuu** [reply]
Joplin needs to be built, Mealie too, Dawarich too. These are some examples

**Reply by MickLesk** [reply]
Yes, that may be. But not much different than some other big scripts ^^
